media_driver/agnostic/common/codec/kernel/merge.py

```diff
@@ -30,9 +30,16 @@
     help='list of input kernels to merge into common kernel (.dat files, order is important)')
 args = parser.parse_args()
 
+removeList = []
 for filename in args.dat_files:
-    if not os.path.isfile(filename) or not os.access(filename, os.R_OK):
-        raise Exception("Can't read input: %s" % filename)
+    print("fileName: " + filename)
+    if(not os.path.exists(filename)):
+    #if not os.path.isfile(filename) or not os.access(filename, os.R_OK):
```

Comment:
Why this is commented out? Either remove completely or provide justification.

Reply:
Removed.